### DISCUSSION TITLE
Feature request: lndk as a ssh service option?

### DISCUSSION THREAD

**Discussion by orbitalturtle** [raspiblitz/raspiblitz#4067] [Feature / App Ideas]
Would be awesome to have the option in the ssh menu to be able to set up a lndk service to forward ln onion messages via LND (and eventually payments to offers) :D https://github.com/lndk-org/lndk

Just a thought. I'm not super up to date on the details of how Raspiblitz works under the hood, but if there's anything I can help with, I'd love to

**Comment by rootzoll** [comment]
Hi. If you or someone else reading this like to research/contribute an lndk integration - then I hope the Contributing guide lines will be a good start: https://github.com/raspiblitz/raspiblitz/blob/v1.9/CONTRIBUTING.md

**Reply by orbitalturtle** [reply]
Thanks for the info, I'll give that a look
